src/app/services/preloader.service.ts

```diff
@@ -8,15 +8,14 @@ export class PreloaderService {
 
   constructor() {
     this._element = document.getElementById(this._selector);
+    console.log(`Hello World`, this._element);
   }
 
   public show() {
     this._element.style['display'] = 'block';
   }
 
-  public hide(delay: number = 0) {
-    setTimeout(() => {
+  public hide() {
       this._element.style['display'] = 'none';
-    }, delay);
   }
```

Comment:
Why was this change necessary @iamonuwa ?

Reply:
That `setTimeOut` set to 0 keeps the preloader active without adding the display none style to the dom